Gemfile

```diff
@@ -1,12 +1,13 @@
 source "http://rubygems.org"
 
-ruby '2.1.6'
+# TODO: remove me
+# ruby '2.1.6'
```

Comment:
I trashed these requirements in https://github.com/github/developer.github.com/commit/ebe16d7577c5fc2d4c798284d995a062b39b95c1, and everything seems to be going great!